app/portainer/components/Base/Buttons/AddButton/AddButton.module.css

```diff
@@ -0,0 +1,4 @@
+.add-button {
+  margin-left: 10px;
```

Comment:
why do we need this?

Reply:
Actually copied it from the existing add button in the repo, they all have that margin-left, don't you think we should leave it like that?